### PR TITLE
Mark redis/LS port settings as deprecated

### PR DESCRIPTION
**Pull request by urso**

### CODE REVIEW

**Comment by ruflin** [comment]
Deprecations should be marked as noted here: https://github.com/elastic/docs#additions-and-deprecations

File: libbeat/docs/outputconfig.asciidoc [hunk 1]
```diff
@@ -447,7 +447,7 @@ batches have been written. Pipelining is disabled if a values of 0 is
 configured. The default value is 0.
 
 [[port]]
-===== port
```
Comment (ruflin) [inline libbeat/docs/outputconfig.asciidoc]: Can you use the `deprecated[]` ascii shortcut here?

**Comment by tsg** [comment]
I'd also add an explicit Changelog entry.

File: libbeat/docs/outputconfig.asciidoc (hunk 1 above)
Comment (urso) [inline libbeat/docs/outputconfig.asciidoc]: done

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -69,6 +69,7 @@ https://github.com/elastic/beats/compare/v5.0.0-beta1...master[Check the HEAD di
 ==== Deprecated
 
 *Affecting all Beats*
+- Setting `ports` has been deprecated in redis and logstash outputs. {pull}2620[2620]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: This should be `port` and not `ports`?

**Comment by ruflin** [comment]
LGTM

**Pull request merged**